Offices/Offices.Data/Entities/Office.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Offices.Data.Entities
+{
+    public class Office: BaseEntity
+    {
+        public string Address { get; set; }
+        public string Photo {  get; set; }
```

Comment:
string photo? are you gonna store here serialized file?

Reply:
I thought about url, but how photos and data like that will be stored in project?

Reply:
You can store a link to the storage where the photo will be, but in that case you should rename the field

Reply:
like PhotoURL?